### PR TITLE
Improve project page

### PR DESCRIPTION
**Pull request by c0m1t**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# source: http://127.0.0.1:8000/graphql
-# timestamp: Sun Dec 15 2019 11:59:10 GMT+0330 (Iran Standard Time)
+# source: http://192.168.109.27:8000/graphql
```
Comment (smmoosavi) [inline schema.graphql]: revert this line

File: src/pages/projects-page/ProjectForm.tsx [hunk 2]
```diff
@@ -66,6 +70,10 @@ export function ProjectForm(props: Props) {
   const user = useAuthGuardUser();
   const userIds = useMemo(() => [user.id], [user]);
 
+  const handleDelete = useCallback(() => {
+    onDelete({ projectReviewId: projectReview.project.id });
```
Comment (smmoosavi) [inline src/pages/projects-page/ProjectForm.tsx]: projectReview.id

File: src/pages/projects-page/ProjectForm.tsx [hunk 3]
```diff
@@ -75,26 +83,39 @@ export function ProjectForm(props: Props) {
...
+                <Rating inputLabel={i18n._('Evaluation')} />
               </Box>
             </DictInputItem>
           </Grid>
+          <Grid item xs={12}>
+            <DictInputItem field="text">
+              <LimitedTextAreaInput
+                label={i18n._('Accomplishments')}
+                maxChars={512}
+                rows={4}
```
Comment (smmoosavi) [inline src/pages/projects-page/ProjectForm.tsx]: remove rows

**Comment by c0m1t** [comment]
Closed in favor of #81 and #82

**Pull request closed without merging**